x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/50_ip.yml

```diff
@@ -18,18 +18,20 @@ setup:
                 runtime_type: ip
                 script:
                   source: |
-                    String m = doc["message"].value;
-                    int end = m.indexOf(" ");
-                    emit(m.substring(0, end));
+                    Matcher m = /([^ ]+) .+/.matcher(doc["message"].value);
+                    if (m.matches()) {
+                      emit(m.group(1));
+                    }
```

Comment:
out of curiosity, what happens when matches returns false for a doc? the runtime field wont have any value thats it?